Unit 8 - 3D Vision ... -> Section Introduction -> Sub Sections History and Applications (#115)
johko/computer-vision-course: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/applications.mdx

> # Applications

Reviewer: ```suggestion
# Applications of 3D Vision
```

> 3D computer vision enables machines to see and understand their environment in three dimensions unlocking a diverse set of applications across numerous industries. Here are some of the exciting applications of 3D computer vision:

Reviewer: ```suggestion
3D computer vision enables machines to see and understand their environment in three dimensions unlocking a diverse set of applications across numerous industries. In this section you can find some of the exciting applications of 3D computer vision.
```
